### PR TITLE
Follow-up edits to PR#2596

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/2596

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @bfallonf @tpoitras please peer review :bow:

File: admin_guide/multiproject_quota.adoc [hunk 1]
```diff
@@ -13,17 +13,25 @@ toc::[]
...
 
 Projects can be selected based on either annotation selection, label selection, or both.
-For example, `oc create clusterquota for-user --project-annotation-selector openshift.io/requester=<user-name> --hard pods=10 --hard secrets=20`
-creates: 
+For example:
+
+====
+----
+$ oc create clusterquota for-user --project-annotation-selector openshift.io/requester=<user-name> --hard pods=10 --hard secrets=20
+----
```
Comment (adellape) [inline admin_guide/multiproject_quota.adoc]: Maybe break this up w/ backslashes? E.g.,

```
$ oc create clusterquota for-user \
    --project-annotation-selector openshift.io/requester=<user-name> \
    --hard pods=10 \
    --hard secrets=20
```

File: admin_guide/multiproject_quota.adoc [hunk 2]
```diff
@@ -59,25 +67,32 @@ status:
...
+<3> A label selector that can be used to select projects.
+<4> A per-namespace map that describes current quota usage in each selected project.
+<5> The aggregate usage across all selected projects.
 ====
 
-This multi-project quota document will control all projects requested by `"<user-name>"` using the default
-project request endpoint.  He will be limited to ten pods and twenty secrets.
+This multi-project quota document controls all projects requested by
+*<user-name>* using the default project request endpoint. You are limited to ten
+pods and twenty secrets.
```
Comment (adellape) [inline admin_guide/multiproject_quota.adoc]: See "Numbers" wrt spelling out vs numerals:

https://obriend.fedorapeople.org/WritingStyleGuide/#chap-Red_Hat_Technical_Publications-Writing_Style_Guide-N

File: admin_guide/multiproject_quota.adoc [hunk 3]
```diff
@@ -97,9 +112,9 @@ secrets   9 20
...
+== Selection Granularity
 
-Because of the locking consideration when claiming quota allocations, the 
-number of active projects selected by a multi-project quota is an important consideration.
+Due to the locking consideration when claiming quota allocations, the number of
+active projects selected by a multi-project quota is an important consideration.
 Selecting more than 100 projects under a single multi-project quota may have
-deleterious effects on API server responsiveness in those projects.
\ No newline at end of file
+deleterious effects on API server responsiveness in those projects.
```
Comment (adellape) [inline admin_guide/multiproject_quota.adoc]: Maybe s/deleterious/detrimental ?

**Comment by adellape** [comment]
@ahardin-rh Comments inline, otherwise :+1:

File: admin_guide/multiproject_quota.adoc [hunk 4]
```diff
@@ -13,17 +13,25 @@ toc::[]
 
 == Overview
 
-A multi-project quota, defined by a `*ClusterResourceQuota*` object, allows xref:../admin_guide/quota.adoc#admin-guide-quota[quota]
-to be shared across multiple projects.  Resources used in each selected project
-will be aggregated and that aggregate will be used to limit resources across
-all the selected projects.
+A multi-project quota, defined by a `*ClusterResourceQuota*` object, allows
+xref:../admin_guide/quota.adoc#admin-guide-quota[quota] to be shared across
```
Comment (bfallonf) [inline admin_guide/multiproject_quota.adoc]: I believe the plural for quota is quotas, so either:

s/quota/a quota or s/quota/quotas

That does depend on if you can share one quota object over multiple projects, I guess...

**Comment by bfallonf** [comment]
@ahardin-rh Just one comment from me. :star: otherwise!

**Comment by ahardin-rh** [comment]
@adellape @bfallonf Thanks for the peer review. All fixed :fireworks:

**Comment by ahardin-rh** [comment]
Rev history in original PR

**Pull request merged**